uaclient/security.py

```diff
@@ -310,6 +316,25 @@ def get_cves_metadata(self):
             self._cves.append(self.client.get_cve(cve_id=cve_id))
         return self._cves
 
+    def released_packages(self, series: str) -> "Dict[str, Dict[str, str]]":
```

Comment:
I may be missing something here, but this method seem to be not called anywhere. Is that expected ?

Reply:
ahh left over from an earlier iteration on this PR, I'll drop it now. I had thought we needed to read USN data for specific binary package versions instead of source package,  but instead we collect all binary pkgs now in `query_installed_source_pkg_versions` and we process those installed binary pkgs one by one in `prompt_for_affected_packages` without need to extract specific versions from the USN.